source/Octopus.Versioning/Octopus.Versioning.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;net40</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net40</TargetFrameworks>

```

Comment:
Can we just target `netstandard2.0` here instead?  I don't think we need both.

Reply:
That is a good point - I'll change that.